server/src/test/java/org/apache/cassandra/sidecar/cluster/instance/InstanceMetadataImplTest.java

```diff
@@ -39,6 +39,10 @@ class InstanceMetadataImplTest
     private static final String DATA_DIR_2 = "test/data/data2";
     private static final String CDC_DIR = "cdc_dir";
     private static final String STAGING_DIR = "staging_dir";
+    private static final String COMMITLOG_DIR = "commitlog";
```

Comment:
I have used the names used in cassandra.yaml for newly introduced directories. Wanted to use default names used in Cassandra, but I don't mind changing the names if required.

Reply:
we should not, this is the default directory name for the commit log in Cassandra, so it would be incorrect to add `_dir`. See https://github.com/apache/cassandra/blob/trunk/conf/cassandra.yaml#L411

Reply:
>so it would be incorrect to add _dir. 

Wondering why CDC_DIR has "_dir" postfix. As per cassandra.yaml, cdc_raw_directory is `cdc_raw`.

Reply:
maybe because it's test code, but it feels we should align with Cassandra directories

Reply:
I ended up with setting these values to non default values in order to test that InstanceMetaImpl is not using default values when these directories are set explicitly. Using "_dir" like in previous constants.

(I wrote this test class a few days back with the same intention, could not recollect it earlier, my apologies!)